app/models/advance_on_docket_motion.rb

```diff
@@ -13,23 +14,28 @@ class AdvanceOnDocketMotion < CaseflowRecord
 
   scope :granted, -> { where(granted: true) }
   scope :eligible_due_to_age, -> { age }
+  scope :eligible_due_to_appeal, ->(appeal) { where(appeal: appeal).where.not(id: age) }
```

Comment:
Is age referring to the `reason` enum?
```suggestion
  scope :eligible_due_to_appeal, ->(appeal) { where(appeal: appeal).where.not(reason: age) }
```
Line 19 also has `where.not(id: age)`

Reply:
This refers to the "free" rails scope. Shorthand for `.where.not(id: AodMotion.age)`

Reply:
Why is `age` being compared against the `id` column instead of the `reason` column?

Reply:
`.where.not(id: age)` === `.where.not(reason: Constants.AOD_REASONS.age)`

Reply:
Gotcha! TIL... interesting. I still prefer `.where.not(reason: :age)` because it's more intuitive and readable to me. Moving on.

Reply:
Found why I originally did this. There are aod motions with nil reasons. `.where.not(reason: Constants.AOD_REASONS.age)` will not return motions with nil reasons

Reply:
https://thoughtbot.com/blog/activerecord-s-where-not-and-nil
Should there be AOD motions without a reason?

Reply:
Nope, but there are